docs/modules/ROOT/pages/user-guide/advanced-soap-server-topics.adoc

```diff
@@ -0,0 +1,63 @@
+= Advanced SOAP server topics
+
+== Running behind a reverse proxy
+
+SOAP requests aimed towards services running on Quarkus can be routed through proxies that generate additional headers
+(e.g. `X-Forwarded-Host`) to keep information from the client-facing side of the proxy servers that is altered or lost when they are involved.
+In those scenarios, Quarkus can be configured to automatically update information like protocol, host, port and URI reflecting the values in these headers.
+
+TIP: Refer to https://quarkus.io/guides/http-reference[Quarkus HTTP reference] for more details.
```

Comment:
Did you intend to reference:  https://quarkus.io/guides/http-reference#reverse-proxy ?

Reply:
No I didn't but the URL with the anchor is much better :-)